test/00_tensor/ViewTests.cu

```diff
@@ -469,6 +469,56 @@ TYPED_TEST(ViewTestsFloatNonComplexNonHalf, Random)
 }
 
 
+
+TYPED_TEST(ViewTestsIntegral, Randomi)
+{
+  MATX_ENTER_HANDLER();
+  {
+    using TestType = cuda::std::tuple_element_t<0, TypeParam>;
+
+    // example-begin randomi-test-1
+    index_t count = 50;
+
+    tensor_t<TestType, 3> t3f({count, count, count});
+    TestType minBound = std::numeric_limits<TestType>::min(); 
+    TestType maxBound = std::numeric_limits<TestType>::max(); 
+    
+    (t3f = (TestType)0).run(this->exec);
+    (t3f = randomi<TestType>({count, count, count}, 0, minBound, maxBound )).run(this->exec);
+    // example-end randomi-test-1   
+    this->exec.sync();
+
+    for (index_t i = 0; i < count; i++) {
+      for (index_t j = 0; j < count; j++) {
+        for (index_t k = 0; k < count; k++) {
+          TestType val = t3f(i, j, k); 
+          ASSERT_NE(val, 0); //bounds are (min,max], so even if min == 0, it shouldn't appear in the data
```

Comment:
I would expect it to be [0,N). What's the reason 0 can't be included?

Reply:
This is the distribution of the [curand_uniform](https://docs.nvidia.com/cuda/curand/device-api-overview.html#distributions) API, and since we're using that under the hood, not much we can do about it. I also think it would  be weird to bias the distribution differently than curand natively does, since we say we're using it under the hood.

Reply:
I need to remove this check anyways though, as it was failing due to the distributions covering 0, so this line and the comment is being removed.